qiskit/pulse/library/parametric_pulses.py

```diff
@@ -208,14 +221,16 @@ class GaussianSquare(ParametricPulse):
         risefall + width <= x < duration
 
         f(x) = amp * exp( -(1/2) * (x - (risefall + width)/2)^2 / sigma^2) )
+
     """
 
     def __init__(
         self,
         duration: Union[int, ParameterExpression],
         amp: Union[complex, ParameterExpression],
         sigma: Union[float, ParameterExpression],
-        width: Union[float, ParameterExpression],
+        width: Union[float, ParameterExpression] = None,
+        risefall_to_sigma: Union[float, ParameterExpression] = None,
```

Comment:
I wonder if `risefall_over_sigma` or `risefall_sigma_ratio` are clearer?

Reply:
perhaps risefall_to_sigma_ratio?

Reply:
On second thought `risefall_to_sigma` is just as good as other suggestions 😄